w3-provider.md

```diff
@@ -148,7 +147,7 @@ A user MAY get a "free" storage provider (from web3.storage) by invoking a self-
 
 Providers MAY impose certain requirements that resource (`with`) must meet.
 
-> Free provider (from web3.storage) requires that resource MUST be an [account][] identifier. Lite and Expert providers (from web3.storage) additionally require that the invocation resource have a payment provider (for billing purposes).
+> Free provider (from web3.storage) requires that resource MUST be a [`did:mailto`] identifier. Lite and Expert providers (from web3.storage) additionally require that the invocation resource have a payment provider (for billing purposes).
```

Comment:
```suggestion
> Free provider (from web3.storage) requires that resource MUST be a [`did:mailto`] identifier. Lite and Expert providers (from web3.storage) additionally require that the invocation resource have a payment method (for billing purposes).
```

(would avoid 'payment provider' so its not confused with 'provider' in the spec itself)

Reply:
> would avoid 'payment provider' so its not confused with 'provider' in the spec itself

The thing is I actually meant that space should have "payment provider"